lib/govuk_component/notification_banner_helpers.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+module GovukComponent
+  module NotificationBannerHelpers
+    # rubocop:disable Metrics/MethodLength
+    def govuk_notification_banner(header = nil, content = nil, tag_options = {}, &block)
+      govuk_notification_banner_options(tag_options)
+
+      header_element = tag.h2(header, class: 'govuk-notification-banner__title', id: 'govuk-notification-banner-title')
+
+      content_element = if block
+                          capture(&block)
+                        else
+                          tag.p(content, class: 'govuk-notification-banner__heading')
+                        end
```

Comment:
not a  blocker and needs tests but this should be extractable to a method . This may handle the Metrics/MethodLength cop violation.

```
def capture_or_tag(content = nil, &block)
  if block
     capture(&block)
  else
    tag.p(content, class: 'govuk-notification-banner__heading')
  end
end
```

And call

``` 
content_element = capture_or_tag(content, &block)
```